Extensions/ParticleSystem/particleemitterobject.ts

```diff
@@ -732,6 +732,10 @@ namespace gdjs {
       }
     }
 
+    getNbParticles() {
```

Comment:
This is the proper fix :) Though, can we name this `getParticleCount`, like in the renderer?
`getNbParticles` was not a very nice name, better change it.

Reply:
(the action can keep its name, just change the function name :))